core/src/main/kotlin/riven/core/service/entity/type/EntityTypeService.kt

```
@@ -119,137 +119,348 @@ class EntityTypeService(
-            type.schema
-        )
-
-        if (breakingChanges.any { it.breaking }) {
-            val existingEntities = entityRepository.findByOrganisationIdAndTypeId(organisationId, type.id)
-            val validationSummary = entityValidationService.validateExistingEntitiesAgainstNewSchema(
-                existingEntities,
-                type.schema,
-            )
+    @Transactional
+    @PreAuthorize("@organisationSecurity.hasOrg(#organisationId)")
+    fun saveEntityTypeDefinition(
```

detekt (1.23.8)

[warning] The function saveEntityTypeDefinition appears to be too complex based on Cyclomatic Complexity (complexity: 25). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)